internal/serve/httphandler/disbursement_handler.go

```
@@ -65,12 +65,19 @@ func (d DisbursementHandler) validateRequest(req PostDisbursementRequest) *valid
 	)
 	v.CheckError(utils.ValidateNoHTML(req.ReceiverRegistrationMessageTemplate), "receiver_registration_message_template", "receiver_registration_message_template cannot contain HTML, JS or CSS")
 	if !req.RegistrationContactType.IncludesWalletAddress {
-		v.Check(
-			slices.Contains(data.GetAllVerificationTypes(), req.VerificationField),
-			"verification_field",
-			fmt.Sprintf("verification_field must be one of %v", data.GetAllVerificationTypes()),
-		)
 		v.Check(req.WalletID != "", "wallet_id", "wallet_id is required")
+		var wallet *data.Wallet
+		if fetchedWallet, err := d.Models.Wallets.Get(ctx, req.WalletID); err == nil {
```

Should we return a validation error if there's an error getting the wallet?